src/compiler/checker.ts

```diff
@@ -91,6 +93,9 @@ module ts {
         var globalBooleanType: ObjectType;
         var globalRegExpType: ObjectType;
         var globalTemplateStringsArrayType: ObjectType;
+        var globalPromiseType: ObjectType;
+        var globalIPromiseType: ObjectType;

```

Comment:
I think if it's IPromise in lib, then this can be globalIPromiseType, I was more talking about the name in lib.